ocp_install_env.sh

```diff
@@ -108,6 +108,7 @@ EOF
 cat <<EOF
     bootstrapExternalStaticIP: "${BOOTSTRAP_IP}"
     bootstrapExternalStaticGateway: "${PROVISIONING_HOST_EXTERNAL_IP}"
+    bootstrapExternalStaticDNS: "192.168.111.1"
```

Comment:
This has to be derived from some variables, not hardcoded (see other parameters).

Reply:
done